stdlib/2/formatter.pyi

```diff
@@ -0,0 +1,104 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/formatter.py
+from typing import Any, IO, List, Optional, Tuple
+
+AS_IS = None
+FONT_TYPE = Tuple[str, bool, bool, bool]
```

Comment:
FONT_TYPE and STYLES_TYPE don't exist at runtime; they should be defined as private. Also, preferably they shouldn't be all upper case, so `_FontType` and `_StylesType`.